### ISSUE TITLE
How to add JavaScript into epub

### ISSUE THREAD

**Issue by Ijaycool** [FolioReader/FolioReader-Android#144]
Am sorry if this isn't the right place to ask for help. Am joining today. 
I had my ebook as HTML file (within a webview app) with a tagger script in the <head> of my HTML file. The script automatically adds link tags to famous names, these links shows a dialogue box onclick, with a brief bio of the famous person.
Is there anyway I can put that JavaScript in this reader so it is injected into the ebook when they are opened in the FolioReader app?
Thanks

**Comment by Ijaycool** [comment]
Meant to say
*with a tagger script in the head tag of my HTML file.

**Comment by Ijaycool** [comment]
No help?

**Comment by mahavir155** [comment]
@Ijaycool i don't think, there is anyway to inject your own javascript into the reader.. but if you add the script in epub html, i think, it should call javascript function to display dialog box..

**Comment by Ijaycool** [comment]
@mahavir155 thanks for responding.
Let me clear, if there is js in my HTML (zipped as epub) FolioReader will call the js function?
If that is correct, then that's amazing.

**Comment by mahavir155** [comment]
@Ijaycool Yes, i think it should call.. we don't remove existing JS functions.. we just inject some JS.. Can you please try if you have just epub and let us know if it work?

**Comment by Ijaycool** [comment]
@mahavir155 thanks.
I tried, FolioReader ignored my js. Can't I hardcode the js into the reader?
Since it's the single ebook I wish to access.
Thanks
